src/main/resources/mycore-classifications/mir_licenses.xml

```diff
@@ -13,8 +13,29 @@
       <label xml:lang="en" text="Open Access related to alliance or national license" />
       <label xml:lang="de" text="Open-Access im Zuge einer Allianz- bzw. Nationallizenz" />
     </category>
+    <category ID="publicdomain">
+      <label xml:lang="en" text="Public Domain Licenses" />
+      <label xml:lang="de" text="Public Domain Lizenzen" />
+      <label xml:lang="x-group" text="true" />
+      <category ID="cc_pdm_1.0">
+        <label xml:lang="de" text="Public Domain Mark 1.0" description="Kein Urheberrechtsschutz" />
+        <label xml:lang="en" text="Public Fomain Mark 1.0" description="No Copyright"/>
+        <label xml:lang="x-dnb-cc" text="cc:cc-pdm-1.0"/>
+        <url xlink:href="https://creativecommons.org/publicdomain/mark/1.0/" xlink:type="locator" />
+      </category>
+      <category ID="cc_0_1.0">
```

Comment:
Should be `cc_zero_1.0` far backwards compatibility with the behavior in [`mods-utils.xsl`](https://github.com/MyCoRe-Org/mycore/blob/775fc0055e656fe4ba0bbb1931cbee1cf67ddf91/mycore-mods/src/main/resources/xsl/mods-utils.xsl#L312).

Reply:
renamed new categories to cc_zero_1.0 and cc_mark_1.0